app/core/admin.py

```diff
@@ -123,6 +124,14 @@ class RecurringEventAdmin(admin.ModelAdmin):
     )
 
 
+@admin.register(PracticeArea)
+class PracticeArea(admin.ModelAdmin):
```

Comment:
Just a nitpick. This should be PracticeAreaAdmin, just to follow the convention.

There are other classes in this file that missed this. I can correct all of them separately too.